.azure-pipelines/ultimate-pipeline.yml

```diff
@@ -1105,6 +1108,205 @@ stages:
       condition: succeededOrFailed()
       continueOnError: true
 
+- stage: asan_tests_windows_profiler
+  condition: > 
+    and(
+      succeeded(), 
+      eq(variables['isBenchmarksOnlyBuild'], 'False'),
+      eq(dependencies.generate_variables.outputs['generate_variables_job.generate_variables_step.IsProfilerChanged'], 'True')
+    )
+  dependsOn: [master_commit_id, generate_variables, build_windows_tracer, build_windows_profiler]
+  variables:
+    masterCommitId: $[ stageDependencies.master_commit_id.fetch.outputs['set_sha.master']]
+  jobs:
+  - template: steps/update-github-status-jobs.yml
+    parameters:
+      jobs: [Win]
+
+  - job: Win
+    pool:
+      vmImage: windows-2019
+    timeoutInMinutes: 60 #default value
+    strategy:
+      matrix:
+        x86:
+          platform: x86
+        x64:
+          platform: x64
+    variables:
+      USE_NATIVE_LOADER: true
```

Comment:
This is obsolete and no longer required
```suggestion
```

Reply:
Removed here 05873b3ff5f2b9f9e3ef04abf84010a0e0b781a2

Reply:
Removed here 05873b3ff5f2b9f9e3ef04abf84010a0e0b781a2